src/factors/Point2D.jl

```diff
@@ -18,6 +18,15 @@ function getSample(cfo::CalcFactor{<:PriorPoint2}, N::Int=1)
   return (rand(cfo.factor.Z, N),)
 end
 
+function (s::CalcFactor{<:PriorPoint2})(res::AbstractVector{<:Real}, 	
+                                        meas, 	
+                                        X1)	
+  #	
+  res[1:2] .= meas[1:2] .- X1[1:2]
+
+  nothing	
```

Comment:
we should probably return `res` in these prior cases too.  There is one quirky corner of dispatch for `Mixture` that needs this function.  Likely a better answer possible but this temp fix is okay for now.